src/libsync/account.cpp

```diff
@@ -474,6 +498,13 @@ void Account::setNonShib(bool nonShib)
     } 
 }
 
+void Account::setBundleRequestsIfCapable(bool bundleRequests){
+    _bundleRequests = bundleRequests;
+}
 
+bool Account::bundledRequestsEnabled()
+{
+    return (_bundleRequests &&  _capabilities.bundledRequest()) ? true : false;
```

Comment:
`(condition) ? true : false;`  is not really good style.  You can just right `return condition`